<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>PaintWidget</class>
    <widget class="QWidget" name="PaintWidget">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>960</width>
                <height>640</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>DrawingBoard</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,9">
            <item>
                <layout class="QVBoxLayout" name="paint_layout">
                    <property name="spacing">
                        <number>12</number>
                    </property>
                    <property name="leftMargin">
                        <number>12</number>
                    </property>
                    <property name="topMargin">
                        <number>12</number>
                    </property>
                    <property name="rightMargin">
                        <number>12</number>
                    </property>
                    <property name="bottomMargin">
                        <number>12</number>
                    </property>
                    <item>
                        <widget class="QToolButton" name="shot_btn">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="text">
                                <string>screenshot</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QToolButton" name="reset_btn">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="text">
                                <string>reset</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QToolButton" name="undo_btn">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="text">
                                <string>undo</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QToolButton" name="dot_btn">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="text">
                                <string>dot adapter</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QToolButton" name="color_btn">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="text">
                                <string>color mode</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QToolButton" name="redo_btn">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="text">
                                <string>redo</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QToolButton" name="ocr_btn">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="text">
                                <string>OCR</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item>
                <widget class="QWidget" name="container" native="true"/>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
